site/docs/components/flex-layout/accessibility.mdx

```diff
@@ -0,0 +1,20 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+When building a layout for part of a page using Flex Layout, it is important to make sure you use semantic HTML, which means using the correct HTML elements for their intended purpose as much as possible.
+
+Flex Layout and Flex Item components support this behavior by allowing access to a prop called ”as” that allows you to render the appropriate HTML instead of a plain “div”.
```

Comment:
```suggestion
Flex Layout and Flex Item components support this behavior by allowing access to a prop called `as` that allows you to render the appropriate HTML instead of a plain `“div”`.
```